Documentation/Getting-Started/Configuration-Guide.md

```diff
@@ -1063,6 +1063,19 @@ seconds. The interval is incremented until the value of `max_retry_interval` is
 reached at which point the listener attempts to bind to the interface every
 `max_retry_interval` seconds.
 
+#### `session_track_trx_state`
+
+Enable or disable getting session transation state via session track mechanism. This
```

Comment:
Describing that the parameter offloads the transaction tracking to the server would be good. I would maybe put it like this:
> Enable or disable session transaction state tracking by offloading it to the backend servers. This is only supported by MariaDB versions 10.2 or newer.

Reply:
ok, actually 10.3, because 10.2 do not provide tracker for autocommit;